stdlib/scanf.ml

```diff
@@ -1358,6 +1362,10 @@ fun ib fmt readers -> match fmt with
   | Bool (pad, rest) ->
     let scan _ _ ib = scan_bool ib in
     pad_prec_scanf ib rest readers pad No_precision scan token_bool
+  | Binary (pad, rest) ->
+    (* TODO not sure if need the same treatment as for String here ? *)
```

Comment:
as per the TODO comment, im not sure if I should be following the same treatment as for String e.g. switching on Formatting_lit/Formatting_gen/rest?

Reply:
fair enough, I can take the TODO comment out